modules/ixBidAdapter.js

```diff
@@ -1473,7 +1494,7 @@ function updateMissingSizes(validBidRequest, missingBannerSizes, imp) {
   } else {
     // New Ad Unit
     if (deepAccess(validBidRequest, 'mediaTypes.banner.sizes')) {
-      let sizeList = deepClone(validBidRequest.mediaTypes.banner.sizes);
+      let sizeList = deepClone(validBidRequest.mediaTypes.banner.sizes.filter(size => apIsValidSize(...size)));
```

Comment:
Too much chaining. Better to store `sizes` in variable & then pass to `deepClone()`

Reply:
Additionally, don't we need to update the `currentSizeList` array on line 1492?

Reply:
Not needed

![image](https://github.com/adpushup/Prebid.js/assets/126073485/21bdec75-0936-4a48-abed-a887ae8db5b4)

Reply:
currentSizeList updated when bidder request goes.